data/language/en-GB.txt

```diff
@@ -3845,7 +3845,7 @@ STR_5503    :Enter hostname or IP address:
 STR_5504    :{SMALLFONT}{BLACK}Show multiplayer status
 STR_5505    :Unable to connect to server.
 STR_5506    :Guests ignore intensities
-STR_5507    :Handymen mow grass by default
+STR_5507    :<not used anymore>
```

Comment:
We now use `<removed string - do not use>` throughout. If you find an occurrence of `<not used anymore>`, let me know.